source/tutorial/backend.txt

```diff
@@ -53,22 +53,28 @@ You'll need to have the following before you can begin this tutorial:
   for MongoDB Atlas and create one for free
   <https://account.mongodb.com/account/register?tck=docs_realm>`__.
 
+Also note that this tutorial doesn't use the Deployment Draft workflow. Unless you have disabled 
+Deployment Drafts, you may need to save your changes to a draft and then deploy them manually.
+
```

Comment:
i think this is a good candidate for a `.. note::` admonition. 

also there should be a ref in here for to wherever Deployment Drafts is documented. 

also would be good to note what a deployment draft is in the note since it's a tutorial and presumably the user wouldn't know what a "Deployment Draft" is

Reply:
I think an admonition here would break the reader's flow unnecessarily. I think the Deployment Drafts flow is fairly self-explanatory in the UI and mostly included this info just in case readers aren't familiar with it (I wasn't). Essentially, I didn't want to draw too much attention to something that's not necessarily critical.